### PR TITLE
[netcore] Cleanup + test fixes

### PR DESCRIPTION
**Pull request by filipnavara**
Avoid using `wget` since it's not installed on macOS by default.

### CODE REVIEW

File: netcore/Makefile.am [hunk 1]
```diff
@@ -56,7 +56,7 @@ xtest-%: prepare check-env dl-test-assets
 	echo -n "***************** $* *********************"
 	ln -sf $(CURDIR)/sdk/$(NETCORESDK_VERSION)/System.Text.Encoding.CodePages.dll assets/extracted/$*/
 	cp $(COREFX_BINDIR)/runtime/netcoreapp-OSX-Debug-x64/xunit.console.deps.json assets/extracted/$*/
-	cp $(COREFX_BINDIR)/runtime/netcoreapp-OSX-Debug-x64/{CoreFx.Private.TestUtilities.dll,Microsoft.DotNet.PlatformAbstractions.dll} assets/extracted/$*/
+	cp $(COREFX_BINDIR)/runtime/netcoreapp-OSX-Debug-x64/CoreFx.Private.TestUtilities.dll assets/extracted/$*/
```
Comment (filipnavara) [inline netcore/Makefile.am]: Reverts change from https://github.com/mono/mono/pull/13776#discussion_r271261073

Comment (EgorBo) [reply netcore/Makefile.am]: yeah more recent CoreFX doesn't have this file there (so the rule fails with file not found)

**Comment by filipnavara** [comment]
@monojenkins OS X x64 Android SDK

**Comment by filipnavara** [comment]
@monojenkins build OS X x64 Android SDK

**Comment by marek-safar** [comment]
Thanks!

**Pull request merged**